src/Bridge/Symfony/Routing/ApiLoader.php

```diff
@@ -55,14 +61,16 @@ public function load($data, $type = null)
 
         foreach ($this->resourceNameCollectionFactory->create() as $resourceClass) {
             $resourceMetadata = $this->resourceMetadataFactory->create($resourceClass);
-            $normalizedShortName = Inflector::pluralize(Inflector::tableize($resourceMetadata->getShortName()));
+
+            $shortNamePath = $resourceMetadata->getShortName() ?? '';
```

Comment:
Thing is, the shortname will always be defined through the [`ShortNameResourceMetadataFactory`](https://github.com/api-platform/core/blob/master/src/Metadata/Resource/Factory/ShortNameResourceMetadataFactory.php). We could force an empty string in the `ResourceMetadata` instead of null as a default value so that the type can be enforced here. Thoughts?